parser-typechecker/tests/Unison/Test/Util/Pretty.hs

```diff
@@ -0,0 +1,44 @@
+{-# LANGUAGE ScopedTypeVariables #-}
+module Unison.Test.Util.Pretty where
+
+import EasyTest
+import qualified Unison.Util.Pretty as P
+
+test :: Test ()
+test =
+  scope "util.pretty" . tests $
+  [ scope "commas" $
+    let
+      actual = P.render 80 $ P.commas input
+      expected = "1, 2, 3"
+    in expectEqual actual expected
+  , scope "wrap.commas" $
+     let
+       actual = P.render 80 $ P.wrap $ P.commas input
+       expected = "1 , 2 , 3"
+     in expectEqual actual expected
```

Comment:
Is this actually expected?

Reply:
Was wondering about this (these rather). I realise now that: no, do not use this particular combination as the white-spacing is messed up. Then again there might be value in keeping these "tests for broken usage" for first time contributors, to see that this is the expected behaviour, and not what you should be using. 

Can remove no problem. (I take it your comment means remove the two `wrap.` cases).